python/ray/autoscaler/commands.py

```diff
@@ -29,10 +29,13 @@
 
 from ray.ray_constants import AUTOSCALER_RESOURCE_REQUEST_CHANNEL
 from ray.autoscaler.updater import NodeUpdaterThread
+import ray.autoscaler.command_runner as cmd_runner
```

Comment:
`from ray.autoscaler.command_runner import set_using_login_shells`

The only reason is because cmd_runner is an overloaded term for the codebase (we use it for CommandRunner objects).